modules/network-graph-view.adoc

```diff
@@ -11,6 +11,8 @@ The network graph provides visibility and control over:
 * The allowed network connections, as defined by the Kubernetes network policies.
 * The active communications paths among namespaces and deployments.
 
+In the menu bar, choose the cluster for which to view information and at least one Namespace.
```

Comment:
I see Namespace is capitalized here, but in the previous two lists, it was lowercase. I'm not sure the correct version, but they should be consistent.